File: tests/fixtures/DocComments.java
/**
 * A documented class.
 */
public class DocComments {
    /**
     * JavaDoc on a method.
     * @param name the name
     * @return greeting string
     */
    public String greet(String name) {
        return "Hello, " + name;
    }

    // Not a javadoc comment.
    public void noDoc() {}
}

File: tests/fixtures/DocComments.kt
/**
 * A documented class.
 */
class DocComments {
    /**
     * KDoc on a method.
     * @param name the name
     * @return greeting
     */
    fun greet(name: String): String {
        return "Hello, $name"
    }

    // Not a KDoc comment.
    fun noDoc() {}
}
